packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -685,6 +687,13 @@ export default abstract class SqlIntegration
     activationMetrics.forEach((m) => this.applyMetricOverrides(m, experiment));
     denominatorMetrics.forEach((m) => this.applyMetricOverrides(m, experiment));
 
+    // Apply regression adjustments
+    metric.regressionAdjustmentEnabled = regressionAdjustmentEnabled;
```

Comment:
Why is this override applied here and not where the other overrides are applied (`applyMetricOverrides`)?